src/cr/cube/matrix.py

```diff
@@ -646,8 +646,13 @@ def _baseline(self):
         columns-var question) are included. This ensures that the baseline is not
         distorted by a large number of missing responses to the columns-question.
         """
-        dim_sum = np.sum(self._all_counts, axis=1)[self._valid_rows_idxs]
-        return dim_sum[:, None] / np.sum(dim_sum)
+        # --- uncond_row_margin is a 1D ndarray of weighted total observation count
```

Comment:
count or counts?

Reply:
added "the" weighted total (for each valid row). This form is consistent with other collection description language.